src/VirtoCommerce.InventoryModule.Data/Services/InventorySearchService.cs

```diff
@@ -33,25 +33,24 @@ public virtual async Task<InventoryInfoSearchResult> SearchInventoriesAsync(Inve
             var cacheKey = CacheKey.With(GetType(), nameof(SearchInventoriesAsync), criteria.GetCacheKey());
             return await _platformMemoryCache.GetOrCreateExclusiveAsync(cacheKey, async (cacheEntry) =>
             {
+                using var repository = _repositoryFactory();
```

Comment:
Are there any significant changes except formatting and new `using` here? 
Maybe we could revert this file?

Reply:
It's just SLint warning.

Reply:
Yes, I meant there are no meaningful task-related changes in this file - so no need for the changes here.